### PR TITLE
initialise elm application with routes

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #1

- Create the Elm application which manages three routes (for now): home page, authentication page and 404 page.

- Testing the different endpoint are parsed correctly: "/" and "/auth"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# DWYL Elm MVP application
+
+MVP application build with Elm
+
+# Run it
+
+- Clone this repository: `git clone git@github.com:dwyl/app-mvp-elm.git && cd app-mvp-elm`
+- Install elm-live, see https://github.com/wking-io/elm-live
+- Run `elm-live src/Main.elm --pushstate -- --outpute elm.js --optimize` see https://github.com/dwyl/learn-elm/issues/156
+- Visit localhost:8000 to see the application
```
Comment (nelsonic) [inline README.md:10]: consider adding more docs on what the person should _expect_ to see when they boot the App. 💭

File: elm.js [hunk 2]
```diff
@@ -0,0 +1,6099 @@
+(function(scope){
```
Comment (nelsonic) [inline elm.js:1]: @SimonLab do we _want_ to be adding the (_compiled_) `elm.js` file to GitHub? 
is it not redundant if people are using `elm-live` to run it?

**Review by nelsonic — COMMENTED** [review]
@SimonLab setup + routes looks good. 👍 
Just one question about `elm.js` before merging. (_do we need the compiled file in `.git`_ ?)
Thanks. ☀️

File: elm.js (hunk 2 above)
Comment (SimonLab) [reply elm.js:1]: At the moment unfortunately I need this file to be on the commits.
I'm using it on Heroku and I haven't find a way to create this file on compilation with the Heroku static buildpack yet.

Comment (nelsonic) [reply elm.js:1]: oh ... 💭 
The way we did it on OA was to compile it on CI and upload the compiled file to S3. 💡 
But that's overkill for this MVP (_for now_).
OK. let's just merge the compiled file for now.

**Review by nelsonic — APPROVED** [review]
@SimonLab thanks for answering my question regarding the compiled file.
Happy to merge when you assign back to me.
(_we can add more documentation later..._)

File: elm.js (hunk 2 above)
Comment (SimonLab) [reply elm.js:1]: I'm using the same method to deploy Elm as described here: https://github.com/dwyl/elm-pwa-example/issues/3#issuecomment-571941816

**Pull request merged**